src/components/Header/Header.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import classes from './Header.css';
+
+const header = () => {
+  return (
+    <header className={classes.header}>
+      <div className={classes.wrap}>
+        <span className={classes.logo}>netflixroulette</span>
```

Comment:
It's a site logo, so it should be an anchor, which leads to index page
+ move to separate component